osu.Game/Screens/SelectV2/FilterControl.cs

```diff
@@ -209,9 +216,20 @@ protected override void LoadComplete()
             sortDropdown.Current.BindValueChanged(_ => updateCriteria());
             groupDropdown.Current.BindValueChanged(_ => updateCriteria());
             collectionDropdown.Current.BindValueChanged(_ => updateCriteria());
+            collectionsSubscription = realm.RegisterForNotifications(r => r.All<BeatmapCollection>(), (collections, changeSet) =>
```

Comment:
Why do this here? Well all the other "react to changes" operations are here so it felt like the right place to have it.